bolt/include/bolt/Core/MCPlusBuilder.h

```diff
@@ -430,6 +430,12 @@ class MCPlusBuilder {
     return Analysis->isIndirectBranch(Inst);
   }
 
+  bool IsUnconditionalJump(const MCInst &Inst) const {
```

Comment:
We need a proper description in the comments how this is different from `isUnconditionalBranch()`.